src/test/java/org/fundacionjala/sfdc/stepdefinition/AccountSteps.java

```diff
@@ -0,0 +1,145 @@
+import org.fundacionjala.sfdc.pageobjects.accounts.SFANewModifyPage;
+import org.fundacionjala.sfdc.pageobjects.SalesForceMainTabClassic;
+import org.fundacionjala.sfdc.util.CommonActions;
+import org.testng.Assert;
+import java.util.Map;
+
+/**
+ * AccountSteps.java
+ * Class for steps of account.feature.
+ */
+public class AccountSteps {
+
+    /**
+     * Given step.
+     */
+    @Given("^Go to accounts section$")
+    public void goToAccountsSection() {
+        SalesForceMainTabClassic salesForceMainTab;
+        salesForceMainTab = new SalesForceMainTabClassic("a.accountBlock", "li#AllTab_Tab");
+        salesForceMainTab.displayOptions();
+        salesForceMainTab.goToAccounts();
+    }
+
+    /**
+     * When step.
+     */
+    @When("^I press new account button a new account form is displayed$")
+    public void iPressNewAccountButtonANewAccountFormIsDisplayed() {
+        SFAMainPage accountsMainPage = new SFAMainPage();
+        CommonActions.clickElement(accountsMainPage.getNewAccountButton());
+    }
+
+    /**
+     * And step.
+     * @param values map.
+     */
+    @And("^I fill the account name field and press the save button$")
+    public void iFillTheAccountNameFieldAndPressTheSaveButton(final Map<String, String> values) {
+        SFANewModifyPage newAccountPage = new SFANewModifyPage();
+        CommonActions.setValues(values, newAccountPage.fillMethodsToFields());
+        CommonActions.clickElement(newAccountPage.getSaveNewAccountButton());
+    }
+
+    /**
+     * And step.
+     * @param values map.
+     */
+    @Then("^a new account is created$")
+    public void aNewAccountIsCreated(final Map<String, String> values) {
+        SFADetailsPage accountDetail = new SFADetailsPage();
+        Assert.assertEquals(accountDetail.getNewAccountSavedName(), values.get("accountName"));
+    }
+
+    /**
+     * When step.
+     */
+    @When("^I choose an account from recent accounts and I click on Edit account$")
+    public void iChooseAnAccountFromRecentAccountsAndIClickOnEditAccount() {
+        SFAMainPage accountsMainPage = new SFAMainPage();
+        CommonActions.clickElement(accountsMainPage.getAccountNameLink());
+        SFADetailsPage accountsDetailPage = new SFADetailsPage();
+        CommonActions.clickElement(accountsDetailPage.getEditButton());
+    }
+
+    /**
+     * And step.
+     * @param values map.
+     */
+    @And("^I edit the account name field and I press the save button$")
+    public void iEditTheAccountNameFieldAndIPressTheSaveButton(final Map<String, String> values) {
+        SFANewModifyPage modifyPage = new SFANewModifyPage();
+        CommonActions.setValues(values, modifyPage.fillMethodsToFields());
+        CommonActions.clickElement(modifyPage.getSaveNewAccountButton());
+    }
+
+    /**
+     * And step.
+     * @param values map.
+     */
+    @Then("^an account is modified")
+    public void anAccountIsModified(final Map<String, String> values) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "aNewAccountIsCreated" on line 57. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)